cupy/_environment.py

```diff
@@ -2,35 +2,71 @@
 This file must not depend on any other CuPy modules.
 """
 
+import distutils
 import os
 import os.path
+import shutil
 
 
 _cuda_path = None
+_nvcc_path = None
```

Comment:
In `cupy/cuda/_environment.py` both were initialized to `()`, which doesn't make much sense.

Reply:
Ouch, that does make sense: it distinguishes between the two states "uninitialized" and "not found". Will send a PR later...😞

Reply:
Fixed in #3413.